README.md

```diff
@@ -278,6 +278,8 @@ add_swagger_documentation \
 This value is added to the `authorizations` key in the JSON documentation.
 
 
+#### *security_definitions*:
```

Comment:
please remove italics, this indicates only features, which are not re-implemented